### PR TITLE
Add direct go to site functionality

### PR DESCRIPTION
**Pull request by andy-d**

### CODE REVIEW

**Comment by andy-d** [comment]
This addresses issue 310

**Comment by andy-d** [comment]
fixes issue #310

File: room/staff.py [hunk 1]
```diff
@@ -136,6 +136,28 @@ def get(self, id=None):
     return common.Respond(self.request, 'site_list_one', d)
 
 
+class SiteLookup(StaffHandler):
+  def get(self, site_number=None):
+    if site_number is not None:
+      query = ndb_models.NewSite.query(ndb_models.NewSite.number == site_number)
+      results = list(query)
+      if not results:
+        return self.redirect_to('NoSiteFound', bad_site_number=site_number)
```
Comment (babybunny) [inline room/staff.py]: You can just write the not found output here.   Can simplify and scrap the extra handler.

File: room/templates/site_list_one.html [hunk 2]
```diff
@@ -6,7 +6,25 @@
...
+                       style="font-size:24px;"
+                       id="site_number"
+                       size="25"
+                       value="{{entries[0].number}}">
+</h2>
+<script type="text/javascript">
+$("#site_number").keyup(function(event){
+    if(event.keyCode === 13){  // enter key
+        var site_number = $('#site_number').val();
+        console.log("Attempting to load site " +  + " ...");
```
Comment (babybunny) [inline room/templates/site_list_one.html]: +  + looks weird
this line can be removed before merge.

File: room/templates/site_list_one.html [hunk 3]
```diff
@@ -6,7 +6,25 @@
   | View a Site
 {% endblock %}
 {%block about_this_page %}
-<h2>Here's site # {{ entries[0].number }}</h2>
+<h2>Site Number <input type="text"
```
Comment (babybunny) [inline room/templates/site_list_one.html]: I think he wanted this input to be on the header of *every* page.

**Review by babybunny — COMMENTED** [review]
Right idea. A few comments.

File: room/staff.py (hunk 1 above)
Comment (andy-d) [reply room/staff.py]: good call

File: room/templates/site_list_one.html (hunk 3 above)
Comment (andy-d) [reply room/templates/site_list_one.html]: well then

**Review by babybunny — APPROVED** [review]
looks great!   Greg will love you for this.

**Pull request merged**